src/components/policy/policy_external/test/policy_manager_impl_ptu_test.cc

```diff
@@ -51,6 +51,10 @@ using ::testing::_;
 using ::testing::Return;
 using ::testing::ReturnRef;
 
+namespace {
+bool is_notification_needed = true;
```

Comment:
@LitvinenkoIra , please name it like global cons. see comment above.
Check other tests